### ISSUE TITLE
Log difference analysis is limited to 8 KB strings

### ISSUE THREAD

**Issue by vikman90** [wazuh/wazuh#1322]
When computing large log difference, Analysisd shows this error:

```
2018/09/15 12:12:10 ossec-analysisd: ERROR: Event size (21591) too long for diff.
```

Can we set the limit to 64 KB?

Reported in Wazuh Lists: https://mail.google.com/mail/u/1/#inbox/FMfcgxvzKbRHwqzZhDglLxpwHwRmnfld

**Comment by vikman90** [comment]
Hi @YashMangla,

Let me add this issue into the project 3.8 so we will take a look at it!

Best regards.

**Comment by chemamartinez** [comment]
Hi @YashMangla, this issue has been fixed at #2032 and will be released ASAP.

Let me close the issue.

Regards.
